### PR TITLE
FTG-38 | Add inventory item pick up

### PR DESCRIPTION
**Pull request by SaronTetra**

### CODE REVIEW

File: core/inventory/inventory_item.gd [hunk 1]
```diff
@@ -0,0 +1,29 @@
+extends CanvasLayer
+
+@export var item_stack: ItemStack
+@onready var sprite = $Sprite2D
+
+var created = false
```
Comment (SzymCrusader) [inline core/inventory/inventory_item.gd:6]: unused variable

File: core/inventory/inventory_item.gd [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+func _ready():
+	$Sprite2D/Debug.text = str(item_stack)
```
Comment (SzymCrusader) [inline core/inventory/inventory_item.gd:10]: unused

File: core/inventory/slot.gd [hunk 3]
```diff
@@ -1,25 +1,78 @@
...
 @export var item: Item
+@export var item_stack: ItemStack
+@export var inventory: Inventory
 
 @onready var selected_scale: float = 3.5
-@onready var normal_scale: float = 3.0
+@onready var normal_scale: float = 3
 
 @onready var texture: Texture2D
+@onready var inventory_item_scene = preload("res://core/inventory/inventory_item.tscn")
```
Comment (SzymCrusader) [inline core/inventory/slot.gd:14]: unused

File: ui/hotbar.gd [hunk 4]
```diff
@@ -24,9 +23,36 @@ func _unhandled_input(event):
...
+	if event.is_action_pressed("hotbar_8"):
+		Events.hotbar_selected.emit(8)
+	if event.is_action_pressed("hotbar_9"):
+		Events.hotbar_selected.emit(9)
+	if event.is_action_pressed("hotbar_0"):
+		Events.hotbar_selected.emit(10)
+	if event.is_action_pressed("hotbar_-"):
+		Events.hotbar_selected.emit(11)
+	if event.is_action_pressed("hotbar_+"):
+		Events.hotbar_selected.emit()
```
Comment (SzymCrusader) [inline ui/hotbar.gd:39]: something happens?

**Review by SzymCrusader — CHANGES_REQUESTED** [review]

File: ui/hotbar.gd (hunk 4 above)
Comment (SaronTetra) [reply ui/hotbar.gd:39]: right, missing `emit(12)`

**Review by SzymCrusader — APPROVED** [review]

**Pull request merged**